.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Test and release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+    
+jobs:
+  test-and-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '^1.19'
+      - uses: ko-build/setup-ko@v0.6
+      - name: Run tests
+        run: go test -v ./...
+      - name: Login to GCR
+        uses: docker/login-action@v2
+        with:
+          registry: gcr.io
+          username: _json_key
+          password: ${{ secrets.GCR_JSON_KEY }}
+      - name: Release
+        run: |
+          build_tag="${{ github.ref_name }}"
+          if [[ "$build_tag" != *-alpha ]]; then
```

Comment:
`alpha` by itself doesnt follow semver.  
is it gonna be used?

Reply:
It actually does acc. to this doc: https://semver.org/#spec-item-9. However we will not normally use it, I will just use it to test this release wf right after the merge.

Reply:
you are right.  i glossed over it and only saw the other examples: 1.0.0-alpha.1, 1.0.0-alpha+001.